### DISCUSSION TITLE
Need a third Button. Is this possible?

### DISCUSSION THREAD

**Discussion by cyborgxxl** [orestbida/cookieconsent#306] [Q&A]
Is it possible to add a third button to the 2 existing buttons? I would like to have the settings as an equivalent button.

Thanks!

**Comment by dombyte** [comment]
I added this ```<button type="button" data-cc="c-settings">Show cookie settings</button>``` to the description as a workaround for now.

**Reply by cyborgxxl** [reply]
Ok, thank you. I've done that now too, but it's not the best way for me.

I have modified the cookieconsent.js with a third button an it works. Unfortunately, this solution is not update safe.

**Reply by dombyte** [reply]
True, then you can add your changes via a pull request to contribute to the project.

**Comment by orestbida** [comment]
An optional third button was added in the upcoming v3. Check out #288 for a preview.

**Reply by montelor** [reply]
I look forward to this version like a dog with a bone!
Thanks for your great work!
